lib/connection/pool.js

```diff
@@ -31,6 +31,11 @@ var DESTROYED = 'destroyed';
 
 var _id = 0;
 
+function supportsClusterTime(topology) {
```

Comment:
should there be a test checking behavior in the case where cluster time isn't supported?

Reply:
all existing mock tests use a lower `maxWireVersion`, so essentially "all other tests" are expressing that code path 😄